utils/QRCode.js

```diff
@@ -0,0 +1,16 @@
+import qrcode from 'qrcode';
+
+async function generateBase64Buffer(str) {
```

Comment:
Why we need a buffer here?
And make another function for base64 string only if needed.

Reply:
The base64 string that is generated by qrcode module is in this format: "data:image/png;base64,<base64code>" (URL). If I keep it and push into attachment property of mailgun, it will raise error. I've seen that Mailgun Attachment only accept buffer data